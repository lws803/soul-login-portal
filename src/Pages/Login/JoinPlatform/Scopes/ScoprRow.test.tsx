import React from 'react';
import { render } from '@testing-library/react';

import ScopeRow from './ScoprRow';

describe(ScopeRow, () => {
  it('renders', () => {
    const { container } = render(<ScopeRow title="TITLE" />);
    expect(container).toMatchSnapshot();
  });
});
