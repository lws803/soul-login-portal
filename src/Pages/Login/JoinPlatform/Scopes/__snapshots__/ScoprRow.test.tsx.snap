// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScopeRow renders 1`] = `
<div>
  <div
    class="chakra-stack css-84zodg"
  >
    <svg
      aria-label="Scope checked"
      class="chakra-icon css-1rcwe32"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M12,0A12,12,0,1,0,24,12,12.014,12.014,0,0,0,12,0Zm6.927,8.2-6.845,9.289a1.011,1.011,0,0,1-1.43.188L5.764,13.769a1,1,0,1,1,1.25-1.562l4.076,3.261,6.227-8.451A1,1,0,1,1,18.927,8.2Z"
        fill="currentColor"
      />
    </svg>
    <p
      class="chakra-text css-1rfq100"
    >
      TITLE
    </p>
  </div>
</div>
`;
